live-examples/html-examples/text-content/dt.html

```diff
@@ -0,0 +1,11 @@
+<p>Please use the following paint colors for the new house:</p>
+
+<dl>
+    <dt>Denim (semigloss finish)</dt>
+    <dd>Ceiling</dd>
+    <dt>Denim (eggshell finish)</dt>
+    <dt>Evening Sky (eggshell finish)</dt>
+    <dd>Layer these on the walls using the sponging technique for texture</dd>
+    <dt>Pearlescent Dream (eggshell finish)</dt>
+    <dd>Trim, including baseboards, crown molding, door and window frames, and window sills</dd>
+</dl>
```

Comment:
Formatting nit: files should end with newline character (I know this isn't explicit in the coding standards, but it's very conventional to require this).